topic_test.go

```diff
@@ -167,7 +167,7 @@ func TestTopicReuse(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	if bytes.Compare(msg.GetData(), firstMsg) != 0 {
+	if bytes.Equal(msg.GetData(), firstMsg) {
```

Comment:
should all these not be `!bytes.Equal`?

Reply:
I think it should. From `bytes.Compare` documentation: `The result will be 0 if a==b, -1 if a < b, and +1 if a > b.`

Reply:
yeah, it's fine.